tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature

```diff
@@ -315,3 +315,138 @@ Feature: Sharing files and folders with internal users
 #    And file "lorem.txt" should be listed in shared-with-others page on the webUI
     And as "user2" file "lorem (2).txt" should not exist
     But as "user3" file "lorem (2).txt" should exist
+
+  Scenario: share a file & folder with another internal user with as custom role and then set all custom permissions
+    Given user "user2" has logged in using the webUI
+    When the user shares folder "simple-folder" with user "User One" as "Custom Role" using the webUI
```

Comment:
if this test is about changing permission on the UI, mention it in the scenario text, use API tests for setup and do not test what is already tested